core/src/commonMain/kotlin/com/powersync/db/internal/InternalDatabase.kt

```diff
@@ -2,13 +2,11 @@ package com.powersync.db.internal
 
 import app.cash.sqldelight.db.Closeable
 import com.persistence.PowersyncQueries
-import com.powersync.PsSqlDriver
 import com.powersync.db.Queries
 import com.powersync.persistence.PsDatabase
 import kotlinx.coroutines.flow.Flow
 
 internal interface InternalDatabase : Queries, Closeable {
-    val driver: PsSqlDriver
```

Comment:
I removed this from the interface. Nobody was calling it, and really shouldn't be. Actual db access should be handled entirely by `InternalDatabaseImpl`.